From 646290364835a7fad2a9a32dc320735bb098bb41 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Wed, 14 Dec 2022 22:47:08 +0100
Subject: [PATCH] imx8mm-var-dart: Switch usb1 dr_mode to host

This is needed for usb devices to work correctly.

NOTE: LTE modem only works if connected tot this
port trough a powered USB hub, otherwise it
resets continuously and there are many errors.

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 .../boot/dts/freescale/imx8mm-var-dart-dt8mcustomboard.dts      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mm-var-dart-dt8mcustomboard.dts b/arch/arm64/boot/dts/freescale/imx8mm-var-dart-dt8mcustomboard.dts
index 76472e922bb9..fcb9283c18b2 100644
--- a/arch/arm64/boot/dts/freescale/imx8mm-var-dart-dt8mcustomboard.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mm-var-dart-dt8mcustomboard.dts
@@ -335,7 +335,7 @@ &uart3 {
 };
 
 &usbotg1 {
-	dr_mode = "otg";
+	dr_mode = "host";
 	hnp-disable;
 	srp-disable;
 	adp-disable;
-- 
2.37.2

